scripts/src/main/resources/scripts/command/lazydocker

```diff
@@ -0,0 +1,116 @@
+    then
+      export PATH=${PATH}:"${LAZYDOCKER_HOME}"
+    fi
+    powershell.exe -command 'start powershell {lazydocker}'
+  else
+    ./lazydocker
+  fi
+}
+
+function doSetup() {
+  if [ ! -d "${LAZYDOCKER_HOME}" ] || [ "${1}" != "silent" ]
+  then
+    doDevonCommand docker setup silent
+    local version="${LAZYDOCKER_VERSION}"
+    # Get latest release
+    if [ -z "${version}" ]
+    then 
+      version="0.18.1"
+      doEcho "Getting latest release..."
+      local latestversion
+      latestversion=$(curl -Ls -o /dev/null -w '%{url_effective}' https://github.com/jesseduffield/lazydocker/releases/latest | sed 's/^.*\///' | sed 's/^v//')
+      doVersionCompare "${latestversion}" "${version}"
+      if [ "${?}" = 1 ]
+      then
+        version=${latestversion}
+      fi
+    fi
+    doCheckDockerVersions
+    doInstall "-" "${LAZYDOCKER_HOME}" "lazydocker" "${version}"
+  fi
+}
+
+function doCheckDockerVersions() {
+  local DOCKERENGVERSION
+  DOCKERENGVERSION=$(docker version --format '{{.Client.Version}}' | sed -ne 's/.*\([0-9][0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p')
+  local DOCKERAPIVERSION
+  DOCKERAPIVERSION=$(docker version --format '{{.Client.APIVersion}}' | sed -ne 's/.*\([0-9]\+\.[0-9]\+\).*/\1/p')
+  doVersionCompare "${DOCKERENGVERSION}" "1.3.0"
+  local res1=$?
+  doVersionCompare "${DOCKERAPIVERSION}" "1.25"
+  local res2=$?
+  if [ "${res1}" == "2" ] || [ "${res2}" == "2" ]
+  then
+    if [ "${res1}" == "2" ]
+    then
+      doWarning "\nIt seems as if your docker-engine version is too old for a proper usage of lazydocker.\nPlease upgrade it at least to version 1.3.0"
+      read -r -p "Continue by pressing enter"
+    fi
+    if [ "${res2}" == "2" ]
+    then
+      doWarning "\nIt seems as if your docker-API version is too old for a proper usage of lazydocker.\nPlease upgrade it at least to version 1.25.0"
+      read -r -p "Continue by pressing enter"
+    fi
+  else 
+    doSuccess "\nDocker-Engine and Docker-API versions are sufficient"
+  fi
+  local DOCKERCOMPOSEVERSION
+  DOCKERCOMPOSEVERSION=$(docker-compose version --short)
+  doVersionCompare "${DOCKERCOMPOSEVERSION}" "1.23.2"
+  res3=$?
+  if [ "${res3}" == "2" ] 
+  then
+    doWarning "Please be aware, that if you want to use docker-compose with lazydocker, then you'll need at least version 1.23.2 of docker-compose"
+  fi
+}
+
+LAZYDOCKER_HOME="${DEVON_IDE_HOME}/software/lazydocker"
+
+# CLI
+case ${1} in 
+"help" | "-h")
+  echo "Install lazydocker."
+  echo
+  echo "Arguments:"
+  echo " setup                    install lazydocker on your machine."
+  echo "                          call lazydocker by executing the command 'devon lazydocker'. Call lazydocker --help for details or use lazydocker directly as preferred."
+  echo
+;;
+"setup" | "s" )
+  doEcho "Installing lazydocker..."
+  doSetup
+;;
+"version" | "v" )
+  lazydversion=$(cat "${LAZYDOCKER_HOME}"/.devon.software.version)
+  doInfo "Current version of lazydocker is ${lazydversion}"
```

Comment:
if we want to have this feature in the future for all commandlets, we could simply add it to `functions` so it does not have to be implemented into every commandlet. However, this is just a side-nore for the record. We can file a separate issue for this. Let us get this lazydocker completed and merged.

Reply:
I created issue #847 for this.